app/config.gradle

```diff
@@ -25,4 +25,7 @@ ext {
     mockitoCoreVersion = '1.10.19'
     playServiceAuthVersion='16.0.1'
     shimmerVersion = "0.5.0"
+    jitpackVersion = "1.1"
```

Comment:
There is no library named 'jitpack'

Also, this section id only for dependencies with duplicate versions